troubleshooting.md

```diff
@@ -746,3 +746,139 @@ an Infra container image for CNI-in-slirp4netns must be created.  The
 instructions for building the Infra container image can be found for
 v2.2.1 [here](https://github.com/containers/podman/tree/v2.2.1-rhel/contrib/rootless-cni-infra),
 and for v3.0.1 [here](https://github.com/containers/podman/tree/v3.0.1-rhel/contrib/rootless-cni-infra).
+
+### 29) Container related firewall rules are lost after reloading firewalld
+Container network can't be reached after `firewall-cmd --reload` and `systemctl restart firewalld` Running `podman network reload` will fix it but it has to be done manually.
```

Comment:
Suggest (Don't spoil the suprise ending until you get to the solution!):
The container's network cannot be reached after the commands `firewall-cmd --reload` or `systemctl restart firewalld` are invoked.